### ISSUE TITLE
Fieldsets are shown empty

### ISSUE THREAD

**Issue by djzwerg** [backdrop-contrib/simplenews_content_selection#4]
If you select a node that contains a fieldset its legend will shown up but the content will not.

**Comment by argiepiano** [comment]
How did you add the fieldset to the node?

**Comment by djzwerg** [comment]
In my case the display is set as a horizontal field group with tabs.
![grafik](https://user-images.githubusercontent.com/49948628/213812085-520d7b77-e266-4a14-bd74-413c770d0379.png)

**Comment by argiepiano** [comment]
So, you are using Field Group for this? I don't think this module can handle Field Groups

**Comment by djzwerg** [comment]
Yes I'm using field groups. Is it possible to make a notice in the readme file that there is no support? I don't know if this case is common or special. Maybe it would work as a workaround to define a custom display for that to show the content of the field group in another way. I will check it.

**Comment by argiepiano** [comment]
I'll investigate why this is not working. It may be easy to fix... we'll see

**Comment by djzwerg** [comment]
Ok. I think that's not a high priority issue but thanks in advance 😄

**Comment by djzwerg** [comment]
Well as I see if you send the newsletter the content of the fieldset is shown. It's a bit crazy.

**Comment by argiepiano** [comment]
Question: in your Simplenews content type, what type of text format does your body field use? "Filtered HTML"? If so, that's a problem, as it will not show much of the markup and classes.  Switch the default for your Simplenews content type to Raw HTML and try again.

It can also be css class problem.

**Comment by djzwerg** [comment]
It is RAW HTML. The source code ist empty at this part.
